changelogs/fragments/4647-gconftool2-command-arg.yaml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
```

Comment:
How about changing this to `bugfixes`, and changing the message to something like `properly escape values when passing them to gconftool-2`? I think it would be good to backport this since this at least to stable-4.

Reply:
done